test/test_nestedtensor.py

```diff
@@ -4928,6 +4928,12 @@ def test_layer_norm_with_lengths(
         i.e. a nested tensor with holes, if operating on the ragged dimension.
         """
 
+        # requires_grad = False does not currently work with dynamo tests and throws this error:
+        # AssertionError: SymInts must use SymNodeVariable. If the underlying value is static,
+        # we will create a ConstantVariable and specialize.
+        if torch._dynamo.is_compiling():
```

Comment:
should this also check for `not requires_grad`? or does the comment need to be updated?

If we're always skipping dynamo then I think it would be better to just xfail or skip if dynamo (with an annotation or with a file in test/dynamo_expected_failures or test/dynamo_skips)

Reply:
test might be flaky, but it recently started to throw errors for both `not requires_grad` and `requires_grad` (though it wasn't throwing errors for `requires_grad` before. i'll update the comment and add the appropriate decorator!